go.mod

```diff
@@ -150,6 +163,14 @@ require (
 	sigs.k8s.io/structured-merge-diff/v4 v4.2.3 // indirect
 )
 
+// Code available in https://github.com/inspektor-gadget/bumblebee/tree/gadget
+replace github.com/solo-io/bumblebee => github.com/inspektor-gadget/bumblebee v0.0.0-20230614170934-120a789ac259
+
+replace github.com/cilium/ebpf => github.com/cilium/ebpf v0.0.0-20230406090554-73acad5e3906
```

Comment:
Each time I run `go mod tidy` is updates `github.com/cilium/ebpf` to use `v0.10.0`, I suppose because that's the version used by bumbleble -> https://github.com/inspektor-gadget/bumblebee/blob/fa5fa890590d32334fc9b4bc10cc0b0707699902/go.mod#L6. 

I think the replace is the right way to avoid changing the line in bumblebee. 

@burak-ok do you have further comments on this?